src/histogram.rs

```diff
@@ -445,16 +445,16 @@ impl MetricVecBuilder for HistogramVecBuilder {
     }
 }
 
-/// A Collector that bundles a set of Histograms that all share the
-/// same Desc, but have different values for their variable labels. This is used
+/// A [`Collector`](::core::Collector) that bundles a set of Histograms that all share the
+/// same [`Desc`](::core::Desc), but have different values for their variable labels. This is used
 /// if you want to count the same thing partitioned by various dimensions
 /// (e.g. HTTP request latencies, partitioned by status code and method).
 pub type HistogramVec = MetricVec<HistogramVecBuilder>;
 
 impl HistogramVec {
-    /// `new` creates a new `HistogramVec` based on the provided `HistogramOpts` and
-    /// partitioned by the given label names. At least one label name must be
-    /// provided.
+    /// Create a new [`HistogramVec`](::core::HistogramVec) based on the provided
```

Comment:
I just found `http:/.../prometheus/::core::HistogramVec` is broken.

Reply:
You need latest nightly rust. This syntax is just landed, see https://github.com/rust-lang/rust/issues/43466

Reply:
So which rustc is used on docs.rs?

Reply:
According to [its about page](https://docs.rs/about), it uses the nightly release. Hope it works!

Reply:
It looks like a wrong path, `HistogramVec` does not belong to `core`. Also, please make sure that it does break other links.

Reply:
Oh sorry, it's a mistake. Fixed.